docs/ai/quickstarts/build-mcp-server-publish-nuget.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Quickstart - Create a minimal MCP Server using .NET and publish to NuGet
+description: Learn to create and connect to a minimal MCP server using .NET and publish it to NuGet.
+ms.date: 06/30/2025
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: jondouglas
+ms.author: jondouglas
+---
+
+# Create, connect and publish a minimal MCP server using .NET to NuGet
+
+In this quickstart, you create a minimal Model Context Protocol (MCP) server using the [C# SDK for MCP](https://github.com/modelcontextprotocol/csharp-sdk), connect to it using GitHub Copilot, and publish it to NuGet. MCP servers are services that expose capabilities to clients through the Model Context Protocol (MCP).
+
+## Prerequisites
+
+- [.NET 10.0 SDK or higher](https://dotnet.microsoft.com/download)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [GitHub Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) for Visual Studio Code
+- [NuGet account](https://www.nuget.org/account/login)
+
+## Create the project
+
+1. In a terminal window, install the `dotnet new mcpserver` template:
+
+   ```bash
+   dotnet new install Microsoft.Extensions.AI.Templates::9.6.0-preview.2.25310.2
+   ```
+
+2. Create a new MCP server app with the `dotnet new mcpserver` command:
+
+   ```bash
+   dotnet new mcpserver -n MinimalMcpServer
+   ```
+
+3. Navigate to the `MinimalMcpServer` directory:
+
+   ```bash
+   cd MinimalMcpServer
+   ```
+
+4. Build the project:
+
+   ```bash
+   dotnet build
+   ```
+
+## Configure the MCP server in Visual Studio Code
+
+Configure GitHub Copilot for Visual Studio Code to use your custom MCP server:
+
+1. If you haven't already, open your project folder in Visual Studio Code.
+2. Create a `.vscode` folder at the root of your project.
+3. Add an `mcp.json` file in the `.vscode` folder with the following content:
+
+   ```json
+    {
+    "servers": {
```

Comment:
Indentation looks a little weird on this line. "servers" should be at a deeper level than the first curly